Sources/SourceKitLSP/Swift/SwiftLanguageService.swift

```diff
@@ -449,7 +454,9 @@ extension SwiftLanguageService {
     inFlightPublishDiagnosticsTasks[notification.textDocument.uri] = nil
     await diagnosticReportManager.removeItemsFromCache(with: notification.textDocument.uri)
 
-    self.documentManager.close(notification)
+    orLog("Connection to the editor closed") {
+      try? self.documentManager.close(notification)
+    }
```

Comment:
I think the `orLog` doesn’t do anything here because `documentManager.close` doesn’t actually throw since you’re using `try?`.

```suggestion
    orLog("Getting document manager") {
      try self.documentManager.close(notification)
    }
```